Purchases/Attribution/AttributionFetcher.swift

```diff
@@ -96,22 +96,23 @@ class AttributionFetcher {
         return true
     }
 
+    var authorizationStatus: FakeTrackingManagerAuthorizationStatus {
+        // should match OS availability here: https://rev.cat/app-tracking-transparency
+        guard #available(iOS 14.0.0, tvOS 14.0.0, macOS 11.0.0, *)  else {
```

Comment:
nit: extra space in `)  else {`
```suggestion
        guard #available(iOS 14.0.0, tvOS 14.0.0, macOS 11.0.0, *) else {
```

Reply:
✅